src/java.base/share/classes/java/text/CompactNumberFormat.java

```diff
@@ -193,23 +200,24 @@
  * {@link java.text.DecimalFormat DecimalFormat}
  * for the specified locale is used.
  *
+ * <h3>Rounding</h3>
+ * {@code CompactNumberFormat} provides rounding modes defined in
+ * {@link java.math.RoundingMode} for formatting.  By default, it uses
+ * {@link java.math.RoundingMode#HALF_EVEN RoundingMode.HALF_EVEN}.
+ *
```

Comment:
Any reason for this part to move before the `Parsing` section?

Reply:
I moved it before `Parsing` as it is related to formatting, so its better under the `Formatting` section.

All the other CompactNumberFormat comments should be implemented as well.